src/main/kotlin/app/revanced/patches/youtube/ad/infocardsuggestions/fingerprints/HideInfocardFingerprint.kt

```diff
@@ -0,0 +1,25 @@
+package app.revanced.patches.youtube.ad.infocardsuggestions.fingerprints
+
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import app.revanced.patches.youtube.ad.infocardsuggestions.annotations.HideInfocardSuggestionsCompatibility
+import app.revanced.patches.youtube.ad.infocardsuggestions.resource.patch.HideInfocardSuggestionsResourcePatch
+import org.jf.dexlib2.Opcode
+import org.jf.dexlib2.iface.instruction.WideLiteralInstruction
+
+@Name("info-cards-drawer-header-fingerprint")
+@HideInfocardSuggestionsCompatibility
+@Version("0.0.1")
+object HideInfocardFingerprint : MethodFingerprint(
```

Comment:
Respectively to the other fingerprints this should be given a conscise name as well.